packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -43,21 +42,23 @@ export async function setupWorkspaceOrgType(defaultUsernameOrAlias?: string) {
     const orgType = await getWorkspaceOrgType(defaultUsernameOrAlias);
     setWorkspaceOrgTypeWithOrgType(orgType);
   } catch (e) {
-    telemetryService.sendException('send_workspace_org_type', e.message);
-    switch (e.name) {
-      case 'NamedOrgNotFound':
-        // If the info for a default username cannot be found,
-        // then assume that the org can be of either type
-        setDefaultUsernameHasChangeTracking(true);
-        setDefaultUsernameHasNoChangeTracking(true);
-        break;
-      case 'NoDefaultusernameSet':
-        setDefaultUsernameHasChangeTracking(false);
-        setDefaultUsernameHasNoChangeTracking(false);
-        break;
-      default:
-        setDefaultUsernameHasChangeTracking(true);
-        setDefaultUsernameHasNoChangeTracking(true);
+    if (e instanceof Error) {
```

Comment:
Why are we checking this condition? Is there a case where e is not an Error?

Reply:
Here I'd assume he's making sure the error will have a `name` attribute below. I'm not sure if there's other scenarios this might be repressing. The safe call would be to add a `console.error(e)` before the if, like he's done below (like in AuthInfo on 49).